src/internal/components/tab-trap/focus-lock.tsx

```diff
@@ -0,0 +1,34 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Question about placement of this file, why is it `components/tab-trap/focus-lock` and not simpler `components/focus-lock`

Reply:
Agree -- will update 👍